src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/CommunityAPIClient.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.component
+
+import mu.KLogging
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+
+@Component
+class CommunityAPIClient(
+  @Qualifier(value = "communityApiWebClient") private val communityAPIWebClient: WebClient,
```

Comment:
qualifier is not needed if the var name matches (just the capitalization is off)